### PR TITLE
Solidjs/add tailwind

### PR DESCRIPTION
**Pull request by hdJerry**
# Background
Each kit should have a styling setup and be configured for usage.

# Acceptance
- [x] Add and configure the style methodology choosen for this KIT
- [x] Make sure the style work both in Dev and Production mode

### CODE REVIEW

**Review by vyktoremario — APPROVED** [review]

File: starters/solidjs-tanstack-tailwind/src/index.css [hunk 1]
```diff
@@ -1,3 +1,7 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
 body {
```
Comment (vyktoremario) [inline starters/solidjs-tanstack-tailwind/src/index.css:5]: Do we till need this though?

Comment (hdJerry) [reply starters/solidjs-tanstack-tailwind/src/index.css:5]: Yes, based on the doc

https://tailwindcss.com/docs/guides/solidjs

Comment (hdJerry) [reply starters/solidjs-tanstack-tailwind/src/index.css:5]: oh, will remove that in the next ticket
I don't wanna disrupt the style yet.

**Review by vyktoremario — APPROVED** [review]

File: starters/solidjs-tanstack-tailwind/src/index.css (hunk 1 above)
Comment (hdJerry) [reply starters/solidjs-tanstack-tailwind/src/index.css:5]: But, i think the body will stay, but the code will go.

**Pull request merged**